src/main/java/frc/robot/vision/GyroAngleValues.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.vision;
+
+/**
+ * A record that represents the gyro's angle which is passed into the limelight's network tables, to the robot_orientation_set entry. Its values
+ * were found in limelightlib sourcecode. The limelight is using it to calculate the BotPose2 position.
+ */
+public record GyroAngleValues(double yaw, double yawRate, double pitch, double pitchRate, double roll, double rollRate) {}
```

Comment:
```suggestion
public record GyroAngleValues(Rotation3d angle, Rotation3d angleRate) {}
```

Reply:
I don't support this change. The reason to have that record in the first place is to document in some way what happens in the limelight's `robot_orientation_set`. That's the order of it too. For context, that's how we use it:  
![20241205_15h35m50s_grim](https://github.com/user-attachments/assets/53e3bd1e-35e3-46c8-9ae7-b20f0009447f)

(red because I merged this code)